src/test/souremap_test.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+/// <reference path="../../node_modules/@types/chai/index.d.ts" />
+/// <reference path="../../node_modules/@types/node/index.d.ts" />
+/// <reference path="../../node_modules/@types/mocha/index.d.ts" />
+import * as chai from 'chai';
+import * as dom5 from 'dom5';
+import * as parse5 from 'parse5';
+import * as path from 'path';
+import {Analyzer} from 'polymer-analyzer';
+import {FSUrlLoader} from 'polymer-analyzer/lib/url-loader/fs-url-loader';
+
+import {Bundler} from '../bundler';
+import {Options as BundlerOptions} from '../bundler';
+
+chai.config.showDiff = true;
+
+const assert = chai.assert;
+const matchers = require('../matchers');
+
+suite('Bundler', () => {
+  let bundler: Bundler;
+
+  async function bundle(inputPath: string, opts?: BundlerOptions):
+      Promise<parse5.ASTNode> {
+        // Don't modify options directly because test-isolation problems occur.
+        const bundlerOpts = Object.assign({}, opts || {});
+        if (!bundlerOpts.analyzer) {
+          bundlerOpts.analyzer = new Analyzer(
+              {urlLoader: new FSUrlLoader(path.dirname(inputPath))});
+          inputPath = path.basename(inputPath);
+        }
+        bundler = new Bundler(bundlerOpts);
+        const documents = await bundler.bundle([inputPath]);
+        return documents.get(inputPath)!.ast;
+      }
+
+  suite('Sourcemaps', () => {
+
+    test('Basic', async() => {
+      const doc = await bundle(
+          'test/html/sourcemaps/A.html',
+          {inlineScripts: true, sourcemaps: true});
+      assert(doc);
+
+      const expected = [
```

Comment:
Hm, is there a more semantic way of testing this? As is, this is a test will ensure that the output doesn't change, but it's not a legible or reviewable test of correctness.

A rough idea of the ideal here: 
 - get the range of the `console.log` statement before bundling
 - bundle
 - parse the sourcemap in the output with a standard library 
 - verify that the source map output maps the new location of console.log back to the old location